GRAPHICS/DrawBinkMovie.md

```diff
@@ -5,21 +5,34 @@ ns: GRAPHICS
 
 ```c
 // 0x7118E83EEB9F7238
-void _DRAW_BINK_MOVIE(int binkMovie, float p1, float p2, float p3, float p4, float p5, int r, int g, int b, int a);
+void _DRAW_BINK_MOVIE(int binkMovie, float posx, float posy, float scalex, float scaley, float rotation, int r, int g, int b, int a);
```

Comment:
Looks like parameter names are case sensitive, in other words, if your function parameters don't match your `## Parameters` section, PR validation checks will fail, i.e.

Naming a parameter `posx` when it should be `posX` as defined in the ` ## Parameters` description section.

```
ParseError: A parameter definition was found for a parameter that does not exist. Parameter name: posX
```